onnxruntime/core/optimizer/qdq_transformer/weight_bias_quantization.cc

```diff
@@ -138,6 +138,7 @@ Status WeightBiasQuantization::ApplyImpl(Graph& graph, bool& modified, int graph
           transB = trans_b_iter->second.i();
         }
         expected_axis = transB == 0 ? 1 : 0;
+        axis = (axis == -1) ? 1 : axis;  // normalize for comparison
```

Comment:
can the normalization be done more generally like this?

https://github.com/microsoft/onnxruntime/blob/88f2652dee465faa485cf68cb39c4489f28945bb/onnxruntime/core/providers/common.h#L24-L35

maybe it can also move out of this Gemm-only block.